packages/elements/src/components/ino-dialog/ino-dialog.scss

```diff
@@ -16,9 +20,24 @@
   --dialog-header-height: var(--ino-dialog-header-height, 20%);
   --dialog-content-height: var(--ino-dialog-content-height, 60%);
   --dialog-footer-height: var(--ino-dialog-footer-height, 20%);
+  --dialog-border-radius: var(--ino-dialog-border-radius, unset);
```

Comment:
These new properties should be added to the comment above (`@prop --dialog-header-height: Height of the header`, ...) so we can generate the table in our readme.

Reply:
Okay, thank you. I changed this now.

Otherwise, everything is fine?